frontend/src/components/ProgressBar/index.jsx

```diff
@@ -0,0 +1,3 @@
+import ProgressBar from "./ProgressBar";
```

Comment:
Is this something that we have to build ourselves? It may be easier to borrow the progress bars from `antd` just for future maintenance and aesthetic consistency.

https://ant.design/components/progress/

They've got a few selections in there that just require passing in a percentage. lmk your thoughts on those and if they satisfy our needs

Reply:
Looks good, added